core/src/test/java/org/apache/calcite/test/RelOptRulesTest.java

```diff
@@ -6297,7 +6297,7 @@ private Sql spatial(String sql) {
             b.field(0),
             b.field(1))
         .sortExchange(
-            RelDistributions.hash(ImmutableList.of(0, 1)),
+            RelDistributions.hashRandom(ImmutableList.of(0, 1), 8),
```

Comment:
Should we add a new test?

Reply:
Make sense. I've updated the pr. Thanks for review.